wordpress-mobile/WordPress-Android
# Add EXPLORE_PLANS quick start task

Fixes #13848

This PR adds a very simple change. If the active task is EXPLORE_PLANS, the focus point is shown on the Plans list item. 

To test:
- Make sure the my site improvements flag is turned on, the app is restarted and your site has quick start in progress
- Click on "Explore plans" QS card
- Notice the app scrolls to the Plans item and there is a focus point visible on the item
- Click on the "Plans" item
- Go back 
- Notice the QS card is now "Done"

![explore_plans](https://user-images.githubusercontent.com/1079756/106759479-18c71e80-6633-11eb-8214-c23609dea80f.gif)

PR submission checklist:
